section4/exercises/classMethods.js

```diff
@@ -28,13 +28,34 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
-};
+  changeProtein() {
```

Comment:
This method should accept one argument, a String, as specified in the directions. The method should re-assign this.protein to the value that was passed in.